src/Microsoft.AspNetCore.Localization/RequestLocalizationOptions.cs

```diff
@@ -109,5 +109,53 @@ public RequestCulture DefaultRequestCulture
         /// </list>
         /// </summary>
         public IList<IRequestCultureProvider> RequestCultureProviders { get; set; }
+
+        
+        /// <summary>
+        /// Adds the set of the supported cultures by the application.
+        /// </summary>
+        /// <param name="cultures">The cultures to be added.</param>
+        /// <returns>The <see cref="RequestLocalizationOptions"/>.</returns>
+        public RequestLocalizationOptions AddSupportedCultures(params string[] cultures)
+        {
+            var supportedCultures = new List<CultureInfo>();
+
+            foreach (var culture in cultures)
+            {
+                supportedCultures.Add(new CultureInfo(culture));
+            }
+
+            SupportedCultures = supportedCultures;
+            return this;
+        }
+
+        /// <summary>
+        /// Adds the set of the supported UI cultures by the application.
+        /// </summary>
+        /// <param name="uiCultures">The UI cultures to be added.</param>
+        /// <returns>The <see cref="RequestLocalizationOptions"/>.</returns>
+        public RequestLocalizationOptions AddSupportedUICultures(params string[] uiCultures)
+        {
+            var supportedUICultures = new List<CultureInfo>();
+            foreach (var culture in uiCultures)
+            {
+                supportedUICultures.Add(new CultureInfo(culture));
+            }
+
+            SupportedUICultures = supportedUICultures;
+            return this;
+        }
+
+        /// <summary>
+        /// Set the default culture to be used by the application when a supported culture could not be determined by
+        /// one of the configured <see cref="IRequestCultureProvider"/>s.
+        /// </summary>
+        /// <param name="defaultCulture">The default culture to be set.</param>
+        /// <returns>The <see cref="RequestLocalizationOptions"/>.</returns>
+        public RequestLocalizationOptions SetDefaultCulture(string defaultCulture)
```

Comment:
What will happen, if the value of the defaultCulture is not listed in the supportedCultures?

Reply:
It will fallback to the `en-US` if i'm not wrong, this is similar to setting the `DefaultRequestCulture` with invalid culture

IMHO it would be nice to write a unit test to cover this case

Reply:
Do you plan to add tests?

Reply:
Of course, this will include the case that you mentioned as well as a few other tests

Reply:
@mkArtakMSFT [this](https://github.com/aspnet/Localization/pull/369/commits/db4cc145224f4415f3e1de03d9b993b96ef90888#diff-90eeb8c3589c85f68d665deef189a6e4R26) is a proven test for the case that you mentioned above